ui/app/src/services/content.ts

```diff
@@ -564,17 +564,26 @@ function insertCollectionItem(
   }
 }
 
-export function uploadDataUrl(site: string, file: any, path: string, xsrfArgumentName: string): Observable<any> {
+function uploadData(
```

Comment:
createFileUpload

Reply:
You changed the name of the wrong method. createFileUpload is this one (the underlying/private abstraction).